eventcore/src/executor/typestate.rs

```diff
@@ -630,6 +630,15 @@ impl<C: Command, ES: EventStore> ExecutionScopeWithState<C, ES> {
         }
         .with_writes(stream_writes))
     }
+    
+    /// Execute command in pipeline style for functional approach
+    pub async fn execute_command_pipeline(
+        self,
+        command: &C,
+        stream_resolver: &mut StreamResolver,
+    ) -> Result<ExecutionScopeWithWrites<C, ES>, CommandError> {
+        self.execute_command(command, stream_resolver).await
+    }
```

Comment:
Not sure what the point of this function is? Isn't it just an alias for the execute_command function?